core/src/main/scala/com/stripe/agate/tensor/Tensor.scala

```diff
@@ -153,6 +155,38 @@ abstract class Tensor[D <: DataType] {
     }
   }
 
+  def nonZero: Tensor[DataType.Int64.type] = {
```

Comment:
Can we add a scala doc comment of what this returns? I am trying to guess but even after reading the test examples I find it confusing. I guessed first it was 1 everywhere the original was nonZero. But that doesn’t seem right.